### PR TITLE
FSET-1201: Content changes for Fast Track

### PR DESCRIPTION
**Pull request by charge-valtech**

### CODE REVIEW

File: app/views/application/personalDetails.scala.html [hunk 1]
```diff
@@ -133,44 +134,33 @@ <h3 class="heading-medium">Qualification requirements</h3>
                   <tr>
                     <td>
                       <ul class="">
-                        @availableSchemes.filter(scheme => !scheme.requiresALevel && !scheme.requiresALevelInStem).map { scheme =>
-                          <li>@scheme.name</li>
-                        }
+                        <li>Business</li>
```
Comment (henricook) [inline app/views/application/personalDetails.scala.html]: @charge-valtech Does this replace automatic scheme listing with hard coded data?

File: app/views/application/personalDetails.scala.html [hunk 2]
```diff
@@ -133,44 +134,33 @@ <h3 class="heading-medium">Qualification requirements</h3>
...
+                      At least 5 GCSEs at grade C and above, or the new grade 4 and above, including English Language and Maths, or <a href="https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels" rel="external" target="_blank">equivalent</a>.
                     </td>
                   </tr>
                   <tr>
                     <td>
                       <ul class="">
-                        @availableSchemes.filter(scheme => scheme.requiresALevel && !scheme.requiresALevelInStem).map { scheme =>
-                          <li>@scheme.name</li>
-                        }
+                        <li>Project delivery</li>
```
Comment (henricook) [inline app/views/application/personalDetails.scala.html]: Same as above, shouldn't we be using the dynamic listings?

Comment (charge-valtech) [reply app/views/application/personalDetails.scala.html]: Ohhh good spot! That's a copy and paste error. The scheme names can be reverted to the dynamic listings, as long as the qualification requirements are left updated.

File: app/views/index/privacy.scala.html [hunk 3]
```diff
@@ -9,18 +9,25 @@ <h2 class="heading-large">Privacy</h2>
...
 	    </p>
 	    <p>When you create an account on the service, the personal information you give us will be used:</p>
 	    <ul>
-	        <li>so you can apply for a Civil Service apprenticeship</li>
+	        <li>so you can apply for a Civil Service Fast Track apprenticeship</li>
 	        <li>so that we can contact you about your application</li>
 	        <li>for administration and research purposes</li>
 	    </ul>
+	    <p>We won't share your personal details for reasons not relating to the apprenticeship.</p>
+		<p>We'll never share your detail with any third party without asking for your permission first.</p>
```
Comment (henricook) [inline app/views/index/privacy.scala.html]: @charge-valtech detail**s**?

Comment (charge-valtech) [reply app/views/index/privacy.scala.html]: Argh, yeah should be details, this was pasted straight from the doc from the content designer, I should've checked it properly.

**Pull request merged**